src/docs/reference/core/Razor/README.md

```diff
@@ -44,6 +44,7 @@ Many extensions methods are available in Razor with `@Orchard`.
 If you want to use an extension method in a view, you can inject an `IOrchardHelper` named `Orchard` at the top of your file:
 
 ```csharp
+@using OrchardCore
```

Comment:
This is right if it's not in the `_ViewImports.cshtml`

Reply:
It's not there by default.